components/permissions/permission_lifetime_manager.h

```diff
@@ -35,7 +37,9 @@ class PermissionLifetimeManager : public KeyedService,
   static void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry);
 
   PermissionLifetimeManager(HostContentSettingsMap* host_content_settings_map,
-                            PrefService* prefs);
+                            PrefService* prefs,
```

Comment:
missing either an include or forward decl for PrefService

Reply:
it's right here: https://github.com/brave/brave-core/pull/8378/files/ced379563c49a79fea8a5393696f5ffe81ca7264#diff-513098d6bab715d3651086dc51d5acb4174242233e56e87d3ff3e31bc682a1deL26